crates/sui-core/src/authority_aggregator.rs

```diff
@@ -95,6 +94,7 @@ impl Default for TimeoutConfig {
 pub struct AuthAggMetrics {
     pub total_tx_certificates_created: IntCounter,
     pub num_signatures: Histogram,
+    pub num_invalid_sigs: Histogram,
     pub num_good_stake: Histogram,
     pub non_retryable_stake: Histogram,
```

Comment:
im not sure if this is a dup metric to track, the `invalid_sig_bad_stake` is a subset of  `non_retryable_stake `